app/components/pages/SubmissionWizard/steps/Files/utils.js

```diff
@@ -0,0 +1,6 @@
+export const errorMessageMapping = {
+  EMPTY: 'Please upload a manuscript.',
+  MULTIPLE: 'Only one file can be uploaded.',
+  UNSUPPORTED: 'That file is not supported.',
+  UNSUCCESSFUL: 'There was a problem uploading your file.',
```

Comment:
"UNSUCCESSFUL" kind of describes all 4 of these situations. Can anyone think of a name which describes this specific scenario, which differentiates it from the others?

Maybe "UPLOAD_UNSUCCESSFUL"? Or "UPLOAD_FAILURE"? Just so it's clear that it's the upload itself that's gone wrong, rather than the user being unsuccessful because the file type is unsupported?

Reply:
`UPLOAD_FAILURE` seems the clearest option to me.

Reply:
I'd be happy with `FAILURE` ... minor point but I think we should group the upload errors ... either with a prefix `UPLOAD_...` or by renaming the object to `uploadErrorMessages` or similar